payments/models.py

```diff
@@ -66,6 +66,11 @@
 
 
 class AbstractBaseProduct(TimeStampedModel, UUIDModel):
+    talpa_ecom_id = models.UUIDField(
```

Comment:
We'd replace talpa_ecom_id with TalpaProductAccounting right?

Reply:
Yeap 👍